lib/rubocop/runner.rb

```diff
@@ -72,7 +72,13 @@ def inspect_files(files)
 
       each_inspected_file(files) { |file| inspected_files << file }
     ensure
-      ResultCache.cleanup(@config_store, @options[:debug]) if cached_run?
+      # OPTIMIZE: Calling ResultCache.cleanup takes time, so we avoid it when
```

Comment:
I'd expand this description to mention that the optimization targets editors forking out to RuboCop.